frb_example/benches/dart/lib/async_benchmark.io.dart

```diff
@@ -0,0 +1,36 @@
+// ignore: unused_import
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:flutter_rust_bridge_benchmark/env/io.dart';
+import 'package:flutter_rust_bridge_benchmark/walltime.dart';
+import 'package:flutter_rust_bridge_benchmark/walltime.io.dart';
+
+import 'async_benchmark.dart';
+
+/// use custom Stopwatch on Dart native
+abstract class AsyncBenchmark extends AsyncBencher {
+  AsyncBenchmark(
+      {required super.name,
+      required super.dylibPath,
+      required super.warmUpTime,
+      required super.measurementTime,
+      required super.sampleSize});
+
+  @override
+  WallTime start() {
+    return AsyncStopWatch()..start();
+  }
+
+  @override
+  Future<void> save(Sample sample) async {
+    final file = File('../../../book/benches/$outputFilename.txt')..create(recursive: true);
+    final content = file.existsSync() ? file.readAsStringSync() : '';
+    if (content.isEmpty) {
+      await file.writeAsString('[${json.encode(sample.toJson())}]');
+    } else {
+      final concatenated = content.replaceFirst(']', ',${json.encode(sample.toJson())}]');
```

Comment:
What is that...? maybe we should do something like:

```
var List<dynamic> contentList = json.decode(content);
file.writeasstring(contentList + sample)

Reply:
Well this is a little innocent trick to append results in an accumulator fashion.
The first time it writes `[{ sample }]`, and subsequently append e.g. `[{ sample01, sample02, sample03, ... }]`.